newa/cli.py

```diff
@@ -114,7 +114,7 @@ def cmd_event(ctx: CLIContext, errata_ids: tuple[str, ...]) -> None:
             releases = ['RHEL-8.10.0', 'RHEL-9.4.0']
```

Comment:
I think we should modify the `else` block here, hardcoding defaults doesn't seem appropriate now when we can fetch details from ET. In particular, in the `else` block we should read event details including `errata_id` from the `init-` file and then fetch details (unifying this part if the block above so we are not duplicating the code)

Reply:
I would propose to use something like this

```
@click.pass_obj
def cmd_event(ctx: CLIContext, errata_ids: tuple[str, ...]) -> None:
    ctx.enter_command('event')

    # when errata_ids were provided on a cmdline
    # process errata_ids loaded from init- files
    if not errata_ids:
        errata_ids = []
        for erratum in ctx.load_initial_errata('init-'):
            errata_ids.append(erratum.event.id)

    for erratum_id in errata_ids:
        event = Event(type_=EventType.ERRATUM, id=erratum_id)

        # fetch erratum details, namely releases
        errata = Erratum.from_errata_tool(event)

        for erratum in errata:
            erratum_job = ErratumJob(event=event, erratum=erratum)

            ctx.save_erratum_job('event-', erratum_job)
```
We should probably add also some error handling when no ids are found at all.

Reply:
I'll fix that.

Reply:
Fixed.